src/cladetime/cladetime.py

```diff
@@ -78,13 +79,54 @@ def __init__(self, sequence_as_of=None, tree_as_of=None):
         else:
             self.url_ncov_metadata = None
 
+    @property
+    def sequence_as_of(self) -> datetime:
+        return self._sequence_as_of
+
+    @sequence_as_of.setter
+    def sequence_as_of(self, date) -> None:
```

Comment:
The logic for setting `sequence_as_of` and `tree_as_of` default values is no longer the same, so created a setter for each of these properties.